src/wallet.js

```diff
@@ -224,21 +224,65 @@ const wallet = {
 
       storage.setItem('wallet:data', dataJson);
 
-      walletApi.getAddressHistory(addresses, (response) => {
```

Comment:
Maybe we should mark `getAddressHistory` as deprecated.

Reply:
You mean keep the old method `getAddressHistory` and mark as deprecated in the docstring?

Reply:
Here, in the lib, we can remove the old method if we'd like to. But on `hathor-core`, we cannot change the API otherwise all wallets will stop working unless they are updated.

Reply:
We are keeping the old API resource and removed the old lib method.